### PR TITLE
[New testcase] PO & PO Member flap status sync across all DBs scenario

### LINKED ISSUE #14984: [Test Gap]: VoQ Chassis: PO Member Flap sync to remote linecards
### Test Gap Description

There was an issue uncovered during the trigger testing: [[Chassis]: PO Member down[PO remains UP] is not handled on remote linecards · Issue #14542 · sonic-net/sonic-buildimage (github.com)](https://github.com/sonic-net/sonic-buildimage/issues/14542)

This was fixed but we currently have GAP in sonic-mgmt tests.

Basically scenario to cover:
Have a multi-member port channel (min-link set to 1) configured on Linecard A, make sure, its synced to Linecard B. 
Flap one of the port from this LAG, and verify the APP-DB, ASIC DB on remote linecard to see that the remote LAG is updated properly(check the asic output).

Also bring down the PO and verify the Port channel on remote linecard.

### Test Plan

Need test case to check PO member flap. Flap the PO member from remote side, and verify PO is updated locally and on all remote linecards.

### Test environment

T2 Chassis

### Attach files (if any)

_No response_

### PR DESCRIPTION
**Pull request by Javier-Tan**
### Description of PR

Summary:
Fixes #14984 and cleanup similar tests in file

I suggest looking at individual commits to have an easier time navigating changes

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Test gap for DB states across local & remote linecards updating for PO member state changes
#### How did you do it?
Added 2 testcases, one where PO went down, one where individual member went down
#### How did you verify/test it?
Tested on T2 testbed
#### Any platform specific information?
T2 VoQ only
#### Supported testbed topology if it's a new test case?
T2 VoQ only

### CODE REVIEW

File: tests/common/helpers/voq_lag.py [hunk 1]
```diff
@@ -60,34 +60,28 @@ def verify_lag_interface(duthost, asic, portchannel, expected=True):
...
+def add_lag(duthost, asic, portchannel=TMP_PC):
+    """Creates a LAG on given ASIC"""
+    logging.info("Adding LAG {} to {} asic{}".format(portchannel, duthost, asic.asic_index))
+    duthost.shell("config portchannel {} add {}".format(asic.cli_ns_option, portchannel))
 
+
+def delete_lag(duthost, asic, portchannel=TMP_PC):
+    """Deletes a LAG on given ASIC"""
+    logging.info("Deleting lag from {}".format(duthost.hostname))
+    duthost.shell("config portchannel {} del {}".format(asic.cli_ns_option, portchannel))
```
Comment (arlakshm) [inline tests/common/helpers/voq_lag.py:72]: use this function iso the shell command https://github.com/sonic-net/sonic-mgmt/blob/9b673bb1f17d201c9698a43720df47aa95d4b0d6/tests/common/devices/sonic_asic.py#L524

File: tests/common/helpers/voq_lag.py [hunk 2]
```diff
@@ -60,34 +60,28 @@ def verify_lag_interface(duthost, asic, portchannel, expected=True):
...
+
+
+def add_members_ip_to_lag(duthost, asic, portchannel_members=None, portchannel_ip=None, portchannel=TMP_PC):
+    """Add members and IP to LAG"""
     if portchannel_members:
+        logging.info("Adding members {} to LAG {}".format(portchannel_members, portchannel))
         for member in portchannel_members:
-            duthost.shell("config portchannel {} member add {} {}"
-                          .format(asic.cli_ns_option, portchannel, member))
+            duthost.shell("config portchannel {} member add {} {}".format(asic.cli_ns_option, portchannel, member))
```
Comment (arlakshm) [inline tests/common/helpers/voq_lag.py:80]: use this function iso the shell command https://github.com/sonic-net/sonic-mgmt/blob/9b673bb1f17d201c9698a43720df47aa95d4b0d6/tests/common/devices/sonic_asic.py#L530

File: tests/common/helpers/voq_lag.py [hunk 3]
```diff
@@ -60,34 +60,28 @@ def verify_lag_interface(duthost, asic, portchannel, expected=True):
...
 
     if portchannel_ip:
-        duthost.shell("config interface {} ip add {} {}"
-                      .format(asic.cli_ns_option, portchannel, portchannel_ip))
+        logging.info("Assigning IP {} to LAG {}".format(portchannel_ip, portchannel))
+        duthost.shell("config interface {} ip add {} {}".format(asic.cli_ns_option, portchannel, portchannel_ip))
```
Comment (arlakshm) [inline tests/common/helpers/voq_lag.py:84]: use the function in sonic_asic.py https://github.com/sonic-net/sonic-mgmt/blob/9b673bb1f17d201c9698a43720df47aa95d4b0d6/tests/common/devices/sonic_asic.py#L517

**Review by arlakshm — COMMENTED** [review]
changes look good. Add a few comments to use the existing functions in sonic_asic.py iso of calling the cli here.

File: tests/common/helpers/voq_lag.py (hunk 1 above)
Comment (Javier-Tan) [reply tests/common/helpers/voq_lag.py]: Resolved in 7b392c5

**Comment by Javier-Tan** [comment]
@arlakshm following up

**Review by arlakshm — APPROVED** [review]

**Pull request merged**